src/components/20-molecules/datepicker/utils/date.js

```diff
@@ -77,9 +84,16 @@ const addLeadingZeroes = (rawNumber, numDigits) => {
   return `${leadingZeroesString}${number}`;
 };
 
-const parseLocalisedDateIfValid = (locale = 'en', inputValue = '') => {
-  if (!Intl.DateTimeFormat.supportedLocalesOf(locale).length) {
-    throw new Error(`locale not supported: ${locale}`);
+const parseLocalisedDateIfValid = (
+  inputLocale = LOCALE_DEFAULT,
+  inputValue = ''
+) => {
+  let locale = inputLocale;
+
+  if (!locale || !Intl.DateTimeFormat.supportedLocalesOf(locale).length) {
+    locale = LOCALE_DEFAULT;
+  } else if (locale.includes('it')) {
```

Comment:
This is incorrect, because proper Italian locale does not format as de-CH! Test for full locale string, as suggested above.

Reply:
The default string for the attribute `local` was `en`. So i thought that both are valid, "en" and "en-GB" or "it" and "it-CH".